src/plugins/crypto/test_key.h

```diff
@@ -0,0 +1,93 @@
+	0x6c, 0x39, 0x76, 0x67, 0x6d, 0x44, 0x45, 0x39, 0x7a, 0x7a, 0x6c, 0x47, 0x6b, 0x6d, 0x35, 0x61, 0x78, 0x49, 0x4e, 0x41, 0x67, 0x44,
+	0x30, 0x43, 0x5a, 0x42, 0x47, 0x7a, 0x5a, 0x66, 0x73, 0x74, 0x71, 0x2f, 0x6b, 0x72, 0x65, 0x66, 0x53, 0x6f, 0x59, 0x44, 0x56, 0x49,
+	0x41, 0x6b, 0x53, 0x32, 0x43, 0x43, 0x4e, 0x41, 0x32, 0x55, 0x6c, 0x0a, 0x2b, 0x36, 0x72, 0x33, 0x49, 0x38, 0x6a, 0x49, 0x62, 0x72,
+	0x31, 0x71, 0x31, 0x47, 0x67, 0x53, 0x4c, 0x30, 0x63, 0x63, 0x6d, 0x61, 0x77, 0x5a, 0x65, 0x63, 0x6b, 0x45, 0x65, 0x58, 0x66, 0x44,
+	0x43, 0x52, 0x73, 0x74, 0x61, 0x79, 0x6f, 0x6f, 0x56, 0x35, 0x4b, 0x4a, 0x5a, 0x58, 0x6e, 0x64, 0x6b, 0x4b, 0x4d, 0x4c, 0x41, 0x66,
+	0x34, 0x6d, 0x48, 0x46, 0x57, 0x68, 0x4c, 0x5a, 0x6a, 0x6a, 0x0a, 0x2b, 0x36, 0x61, 0x65, 0x34, 0x77, 0x67, 0x37, 0x67, 0x31, 0x43,
+	0x6e, 0x46, 0x6f, 0x62, 0x65, 0x51, 0x34, 0x4e, 0x55, 0x73, 0x43, 0x53, 0x59, 0x4e, 0x4d, 0x6d, 0x76, 0x44, 0x4f, 0x69, 0x44, 0x63,
+	0x4b, 0x6e, 0x46, 0x58, 0x41, 0x64, 0x31, 0x6a, 0x45, 0x7a, 0x41, 0x35, 0x58, 0x4d, 0x6d, 0x7a, 0x54, 0x6c, 0x33, 0x6f, 0x52, 0x6b,
+	0x68, 0x50, 0x7a, 0x31, 0x46, 0x47, 0x55, 0x68, 0x61, 0x0a, 0x68, 0x51, 0x57, 0x47, 0x76, 0x51, 0x31, 0x48, 0x6d, 0x53, 0x71, 0x6e,
+	0x6d, 0x2b, 0x61, 0x49, 0x6e, 0x77, 0x51, 0x59, 0x41, 0x51, 0x49, 0x41, 0x43, 0x51, 0x55, 0x43, 0x56, 0x37, 0x69, 0x51, 0x7a, 0x77,
+	0x49, 0x62, 0x44, 0x41, 0x41, 0x4b, 0x43, 0x52, 0x44, 0x61, 0x39, 0x6a, 0x57, 0x78, 0x66, 0x79, 0x4d, 0x4f, 0x6a, 0x55, 0x65, 0x6c,
+	0x41, 0x2f, 0x39, 0x70, 0x35, 0x79, 0x69, 0x32, 0x0a, 0x38, 0x43, 0x4d, 0x2b, 0x63, 0x2f, 0x33, 0x59, 0x77, 0x6e, 0x70, 0x4e, 0x49,
+	0x78, 0x4a, 0x77, 0x73, 0x69, 0x76, 0x61, 0x2f, 0x73, 0x78, 0x69, 0x2b, 0x65, 0x4d, 0x61, 0x38, 0x6c, 0x6d, 0x5a, 0x45, 0x5a, 0x54,
+	0x4a, 0x44, 0x4b, 0x2f, 0x63, 0x77, 0x65, 0x41, 0x61, 0x79, 0x55, 0x6c, 0x61, 0x35, 0x44, 0x58, 0x6c, 0x41, 0x2b, 0x72, 0x43, 0x72,
+	0x45, 0x6c, 0x54, 0x44, 0x79, 0x6c, 0x58, 0x0a, 0x39, 0x6a, 0x58, 0x68, 0x62, 0x44, 0x5a, 0x68, 0x6b, 0x66, 0x69, 0x57, 0x6a, 0x6b,
+	0x69, 0x62, 0x43, 0x50, 0x73, 0x75, 0x59, 0x2b, 0x37, 0x78, 0x2f, 0x6e, 0x65, 0x4f, 0x5a, 0x6e, 0x49, 0x4f, 0x31, 0x56, 0x74, 0x50,
+	0x73, 0x32, 0x34, 0x76, 0x53, 0x36, 0x6b, 0x39, 0x34, 0x71, 0x34, 0x6c, 0x53, 0x67, 0x43, 0x2f, 0x52, 0x7a, 0x4e, 0x32, 0x66, 0x62,
+	0x79, 0x76, 0x73, 0x70, 0x51, 0x57, 0x0a, 0x6e, 0x4c, 0x51, 0x34, 0x36, 0x43, 0x36, 0x31, 0x59, 0x50, 0x67, 0x51, 0x78, 0x61, 0x51,
+	0x51, 0x54, 0x68, 0x57, 0x6b, 0x35, 0x30, 0x74, 0x2f, 0x45, 0x2f, 0x48, 0x32, 0x69, 0x41, 0x37, 0x38, 0x6b, 0x43, 0x58, 0x52, 0x62,
+	0x51, 0x3d, 0x3d, 0x0a, 0x3d, 0x78, 0x43, 0x41, 0x38, 0x0a, 0x2d, 0x2d, 0x2d, 0x2d, 0x2d, 0x45, 0x4e, 0x44, 0x20, 0x50, 0x47, 0x50,
+	0x20, 0x50, 0x52, 0x49, 0x56, 0x41, 0x54, 0x45, 0x20, 0x4b, 0x45, 0x59, 0x20, 0x42, 0x4c, 0x4f, 0x43, 0x4b, 0x2d, 0x2d, 0x2d, 0x2d,
+	0x2d, 0x0a
+};
+static const size_t test_key_asc_len = 1960;
```

Comment:
Yes, I took over the output of `xxd -i` but I can adapt to `sizeof(test_key_asc)` if you prefer.